x-pack/plugins/security_solution/public/common/components/link_to/index.ts

```diff
@@ -43,3 +44,18 @@ export const useFormatUrl = (page: SecurityPageName) => {
   );
   return { formatUrl, search };
 };
+
+export const useTimelineUrl = () => {
```

Comment:
I feel like we do not need this hook since we already have `useFormatUrl` who can achieve all of that